charts/hlf-k8s/requirements.yaml

```diff
@@ -14,18 +14,22 @@
 
 dependencies:
   - name: hlf-ca
-    repository: https://kubernetes-charts.storage.googleapis.com/
+    repository: https://owkin.github.io/charts/
     version: 1.2.0
     condition: hlf-ca.enabled
   - name: hlf-ord
-    repository: https://kubernetes-charts.storage.googleapis.com/
-    version: 1.4.1
+    repository: https://owkin.github.io/charts/
+    version: 2.0.0
     condition: hlf-ord.enabled
   - name: hlf-peer
-    repository: https://kubernetes-charts.storage.googleapis.com/
-    version: 1.6.1
+    repository: https://owkin.github.io/charts/
+    version: 2.0.0
+    condition: hlf-peer.enabled
+  - name: hlf-couchdb
+    repository: https://owkin.github.io/charts/
+    version: 1.0.7
     condition: hlf-peer.enabled
   - name: nginx-ingress
-    repository: https://kubernetes-charts.storage.googleapis.com/
+    repository: https://charts.helm.sh/stable
```

Comment:
I think we need a PR in substra-tests to add `https://charts.helm.sh/stable` and `https://owkin.github.io/charts/`

Reply:
Yes :) 
https://github.com/SubstraFoundation/substra-tests/pull/157